rts/Lua/LuaSyncedRead.cpp

```diff
@@ -386,6 +386,8 @@ bool LuaSyncedRead::PushEntries(lua_State* L)
 	REGISTER_LUA_CFUNC(TraceRayGroundInDirection);
 	REGISTER_LUA_CFUNC(TraceRayGroundBetweenPositions);
 
+	REGISTER_LUA_CFUNC(IsFeatureVisible);
```

Comment:
would be nice to put this alongside other feature-related functions

Reply:
Done!